packer/aws/variables.pkr.hcl

```
@@ -17,7 +17,7 @@ variable "base_ami_name" {
 variable "rke2_version" {
   type        = string
   description = "RKE2 version to install on the AMI"
-  default     = "v1.26.7+rke2r1"
+  default     = "v1.27.5+rke2r1"
```

```suggestion
  default     = "v1.26.7+rke2r1"
```
Forgot I changed this, going to check if this was the issue with CI...if so will open a ticket to track updating to 1.27.x